### PR TITLE
#162376432 Add component for displaying error messages

### PR DESCRIPTION
**Pull request by ssewilliam**
#### What does this PR do?
- Add a page not found the component 

#### Description of Task to be completed?
- Add component for displaying an error message when a user tries to access wrong routes

#### How should this be manually tested?
- Setting up the backend
     - Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
     - Enter into the project directory - `cd travel_tool_back`
     - checkout the branch git checkout `develop`
     - Install dependencies using `yarn install`
     - Rollback and run database migrations with `yarn db:rollmigrate`
     - start the backend server with `yarn run start:dev`

- Setting up the frontend
   - Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
   - Enter into the project directory - `cd travel_tool_front`
   - Checkout to the branch `ft-404-error-handlers-162376432` then run the app `yarn start`
   - Login to the application
   - Navigate to the `/fake_url` or any unknown route on the application
   - It is expected that the page not found component will be rendered.

#### Any background context you want to provide?
- All routes that are not configured in the `Routes` will render this new component

#### What are the relevant pivotal tracker stories?
[#162376432](https://www.pivotaltracker.com/story/show/162376432)

#### Screenshots (if appropriate)
screen shot 2018-12-06 at 23 53 40

### CODE REVIEW

**Comment by ssewilliam** [comment]
Thanks for the feedback @ezrqnkemboi, I will be working on [another task](https://www.pivotaltracker.com/story/show/162474402) for the second part of your requested change.
For now, let me work on fixing the /link and using BEM approach on the styles I have added.

**Comment by janetnim** [comment]
The link to go back still takes me to the 404 page. I think this should be to requests page link.

File: src/views/ErrorPages/NotFound/index.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    return (
+      <div className="pageOverlay">
+        <div className="notFoundBody">
+          <div className="bg_404" />
+          <p>
+            The page you are looking for might have been removed, had its name
+            changed or is temporarily unavailable.
+          </p>
+          <center>
+            <a className="button" href="/link">
```
Comment (janetnim) [inline src/views/ErrorPages/NotFound/index.jsx]: This href should link to requests page. Kindly make the changes.

**Comment by ssewilliam** [comment]
Thanks, @ezrqnkemboi and @janetnim. With the new changes, I have been able to resolve all the issues raised earlier on this PR.
Please review.

**Comment by ezrqnkemboi** [comment]
I like the color change. It is good. The click here to go back takes me to request, which is very wonderful. I have some little concern for some endpoints such as : 
http://local.andela.com:3000/requests/my-verifications, if I add - in the end, the verification modal is opened with undefined values. This is the same case as that of guest house management http://local.andela.com:3000/residence/manage/guest-houses/10948
I may not be clear on this but we can take a look at this. This includes that of CSS p and h2 rectification.

File: src/views/ErrorPages/NotFound/index.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <div className="notFoundBody">
+          <div className="bg_404">
+            <img src={notFoundImage} alt="page not found" /> 
+          </div>
+          <p>
+            The page you are looking for might have been removed, had its name
+            changed or is temporarily unavailable.
+          </p>
+          <center>
+            <a className="button" href="/requests" onClick={this.handleNaviagation}>
```
Comment (ezrqnkemboi) [inline src/views/ErrorPages/NotFound/index.jsx]: Good work.

**Pull request merged**